docs/appendices/compatibility.rst

```diff
@@ -89,19 +88,17 @@ the record is modified. This version number can be used to implement patterns
 like :ref:`sql_occ`, which can be used to solve many of the use cases that
 would otherwise require traditional transactions.
 
+Transaction statements do not throw an error.
+
 Unsupported features and functions
 ==================================
 
-These **features** of standard SQL are not supported:
+These **features** of `standard SQL`_ are not supported:
 
-- Stored Procedures
+- Stored procedures
 
 - Triggers
 
-- ``VALUES`` list used as constant tables
-
-- ``WITH`` Statements
```

Comment:
I think this was referring to [common table expressions](https://www.postgresql.org/docs/current/queries-with.html) which we still do not support.